### PR TITLE
chore(exports): remove unused export from fetch-utils

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
This removes an unused export from src/compiler/sys/fetch/fetch-utils.ts
and inlines a function which was used in only one place. A JSDoc is
added for the function that is changed.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Unit tests (`npm test`) were run locally and passed
- [x] E2E Tests (`npm run test.karma.prod`) were run locally and passed
- [x] Prettier (`npm run prettier`) was run locally and passed

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

GitHub Issue Number: N/A

## What is the new behavior?

This removes an unused export and inlines one function which was used in only one place.

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/compiler/sys/fetch/fetch-utils.ts [hunk 1]
```diff
@@ -2,8 +2,16 @@ import type * as d from '../../../declarations';
...
+ * A fetch wrapper which dispatches to `sys.fetch` if present, and otherwise
+ * uses `global.fetch`.
+ *
+ * @param sys a compiler system object
+ * @param input a `RequestInfo` object
+ * @param init an optional `RequestInit` object
+ * @returns a Promise wrapping a response
+ */
 export const httpFetch = (sys: d.CompilerSystem, input: RequestInfo, init?: RequestInit): Promise<Response> => {
-  console.trace(input);
```
Comment (alicewriteswrongs) [inline src/compiler/sys/fetch/fetch-utils.ts:6]: there's no reason for us to leave this lurking in here, right?

Comment (alicewriteswrongs) [reply src/compiler/sys/fetch/fetch-utils.ts:6]: I hope not anyway!

**Review by alicewriteswrongs — COMMENTED** [review]
I found something kinda funky

File: src/compiler/sys/fetch/fetch-utils.ts (hunk 1 above)
Comment (rwaskiewicz) [reply src/compiler/sys/fetch/fetch-utils.ts:6]: I hope not 😆

**Review by rwaskiewicz — APPROVED** [review]
LGTM!

**Pull request merged**